ops/src/test/scala/test/ammonite/ops/ExampleTests.scala

```diff
@@ -114,8 +114,9 @@ object ExampleTests extends TestSuite{
       val fullInfo = stat.full(wd/"file1.txt")
       fullInfo.ctime: FileTime
       fullInfo.atime: FileTime
-      fullInfo.group: GroupPrincipal
-
+      if (Unix()) {
```

Comment:
@piotrkwiecinski is this really necessary? We are just doing a type-ascription, and not a `asInstanceOf`, so even if `group` is null it shouldn't fail. The only way this can fail is by failing to compile; the runtime value of `fullInfo.group` isn't ever used

Reply:
@lihaoyi I would say it's more like a temporary workaround. Issue comes from https://github.com/lihaoyi/utest/blob/master/utest/shared/src/main/scala/utest/framework/Formatter.scala#L19

It fails on my windows machine, so I'll play around with it more.